dace/transformation/interstate/loop_unroll.py

```diff
@@ -8,10 +8,14 @@
 from dace.properties import Property, make_properties
```

Comment:
There are some changes in this file which seem to be required but I didn't try out if it needs all of them or not. If I reset that file to the state of master I get an undefined symbol error